src/dev-app/slider/slider-demo.html

```diff
@@ -1,88 +1,191 @@
+        <td class="demo-value-cell">{{ngStartThumb3.value}} : {{ngEndThumb3.value}}</td>
+      </tr>
+      <tr>
+        <th class="demo-table-header">Min</th>
+        <td class="demo-control-cell"><input class="demo-slider-control" type="number" (input)="updateMin($event.target)" [value]="min" /></td>
+        <td class="demo-value-cell">{{nativeSlider3.min}}</td>
+        <td class="demo-value-cell">{{ngThumb3.min}}</td>
+        <td class="demo-value-cell">{{ngStartThumb3.min}} : {{ngEndThumb3.min}}</td>
+      </tr>
+      <tr>
+        <th class="demo-table-header">Max</th>
+        <td class="demo-control-cell"><input class="demo-slider-control" type="number" (input)="updateMax($event.target)" [value]="max" /></td>
+        <td class="demo-value-cell">{{nativeSlider3.max}}</td>
+        <td class="demo-value-cell">{{ngThumb3.max}}</td>
+        <td class="demo-value-cell">{{ngStartThumb3.max}} : {{ngEndThumb3.max}}</td>
+      </tr>
+      <tr>
+        <th class="demo-table-header">Step</th>
+        <td class="demo-control-cell"><input class="demo-slider-control" type="number" (input)="updateStep($event.target)" [value]="step" /></td>
+        <td class="demo-value-cell">{{nativeSlider3.step}}</td>
+        <td class="demo-value-cell">{{ngThumb3.step}}</td>
+        <td class="demo-value-cell">{{ngStartThumb3.step}} : {{ngEndThumb3.step}}</td>
+      </tr>
+      <tr>
+        <th class="demo-table-header">Disabled</th>
+        <td class="demo-control-cell"><input class="demo-slider-control" type="checkbox" (input)="updateDisabledState($event.target)" [value]="disabled" /></td>
+        <td class="demo-value-cell">{{nativeSlider3.disabled}}</td>
+        <td class="demo-value-cell">{{ngThumb3.disabled}}</td>
+        <td class="demo-value-cell">{{ngStartThumb3.disabled}} : {{ngEndThumb3.disabled}}</td>
+      </tr>
+    </table>
 
-<mat-tab-group>
-  <mat-tab label="One">
-    <mat-slider min="1" max="5">
-      <input value="3" matSliderThumb>
+    <input class="demo-native-slider" type="range" #nativeSlider3 [value]="value" [min]="min" [max]="max" [step]="step" [disabled]="disabled" />
+    <br>
+    <mat-slider discrete showTickMarks color="{{colorModel}}" [min]="min" [max]="max" [step]="step" [disabled]="disabled">
+      <input matSliderThumb #ngThumb3 [value]="value" />
+    </mat-slider>
+    <br>
+    <mat-slider discrete showTickMarks color="{{colorModel}}" [min]="min" [max]="max" [step]="step" [disabled]="disabled">
+      <input matSliderStartThumb #ngStartThumb3 [value]="value" />
+      <input matSliderEndThumb #ngEndThumb3 />
     </mat-slider>
   </mat-tab>
-</mat-tab-group>
 
-<h1>Range slider</h1>
-<mat-slider min="200" max="500" step="100" discrete showTickMarks>
-  <input value="300" matSliderStartThumb>
-  <input value="400" matSliderEndThumb>
-</mat-slider>
+  <mat-tab label="Two-way property bindings">
+    <table class="demo-data-table">
+      <tr>
+        <th class="demo-table-header">Inputs</th>
```

Comment:
FWIW the demos are view-encapsulated so you don't have to apply classes to everything, you can just target `table` and `th`.